charts/vh-video-web/values.dev.template.yaml

```diff
@@ -10,4 +10,6 @@ java:
         EJUDAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
         EJUDAD__REDIRECTURI: https://${SERVICE_FQDN}/home
         DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
-        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
\ No newline at end of file
+        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
+        VHSERVICES__VIDEOAPIURL: https://vh-video-api-pr-674.dev.platform.hmcts.net/
```

Comment:
just a reminder to revert before merging

Reply:
done